source/Geometry/src/Disk.cc

```diff
@@ -9,8 +9,8 @@ Disk::Disk(int id, double zOffset, double zECCentre, double zECHalfLength, const
  zError(        "zError"     , parsedAndChecked()),
  m_innerRadius( "innerRadius", parsedAndChecked()),
  m_outerRadius( "outerRadius", parsedAndChecked()),
- m_bigDelta(    "bigDelta"   , parsedAndChecked()),
- m_rOverlap(    "rOverlap"   , parsedOnly(), 1.),
+ bigDelta(      "bigDelta"   , parsedAndChecked()),
```

Comment:
The idea behind this was to keep the private variables as e.g. m_rOverlap(), but the public as rOverlap() -> simply applying the coding conventions. The configuration doesn't care about any coding conventions, hence the string counterpart is without m_ . Just let me know, what the procedure should be and I will follow that in the whole code... It's matter of taste... If we want to keep them without m_ I will make all public.